runtime/doc/news.txt

```diff
@@ -0,0 +1,39 @@
+*news.txt*    Nvim
+
+
+			    NVIM REFERENCE MANUAL
+
+
+Notable changes in Nvim 0.9 from 0.8                                     *news*
+
+                                       Type |gO| to see the table of contents.
+
+==============================================================================
+BREAKING CHANGES                                                *news-breaking*
```

Comment:
If this allows us to eliminate https://github.com/neovim/neovim/issues/14090 , that is a strong benefit.

Reply:
I'm not 100% sure on that; the issue has the (significant!) advantage that you can subscribe to it and get notifications of breaking changes instead of needing to check a file.

On the other hand; the issue is growing unbounded, so that's clearly not sustainable long-term (and pruning comments after a release is tedious).

Reply:
Maybe that issue could be closed periodically with link to newly opened breaking-changes issue that interested users could very easily subscribe to? Perhaps they could be closed/recreated per release?

Reply:
Mainly I don't want to paste things in multiple places.

Reply:
Yeah, I was not clear enough: I agree that `news.txt` is a better place for this, all things considered.